implementations/rust/ockam/ockam_app_lib/src/lib.rs

```diff
@@ -1,3 +1,5 @@
+#![recursion_limit = "256"]
```

Comment:
Could you please add an explanation here?

Reply:
The compiler complained loudly about breaking some recursion level at some stage and I had to raise the recursion limit. Strangely this does not happen anymore, so I just removed it.